source/Weapon.cpp

```diff
@@ -330,6 +336,11 @@ void Weapon::LoadWeapon(const DataNode &node)
 			++it;
 		}
 	}
+
+	// Only when the weapon is not safe and has a blast radius is safeRange needed,
+	// except if it is already overridden.
+	if(!isSafe && blastRadius > 0 && !safeRangeOverriden)
+		safeRange = (blastRadius + triggerRadius);
```

Comment:
Why blast radius + trigger radius? The trigger radius doesn't increase the blast range. If anything, I'd think that 2x the blast radius would make more sense as a safe range.